package.json

```diff
@@ -1,57 +1,58 @@
 {
-    "name": "perseo",
-    "description": "IOT CEP front End",
-    "version": "0.4.1-next",
-    "homepage": "",
-    "author": {
-        "name": "crbrox",
-        "email": "carlos.romerobrox@telefonica.com"
-    },
-    "repository": {
-        "type": "git",
-        "url": "git@github.com:telefonicaid/perseo-fe.git"
-    },
-    "bugs": {
-        "url": "https://github.com/telefonicaid/perseo-fe/issues"
-    },
-    "main": "lib/perseo",
-    "engines": {
-        "node": ">=0.10.8"
-    },
-    "scripts": {
-        "test": "grunt test"
-    },
-    "devDependencies": {
-        "chai": "~1.8.0",
-        "grunt": "~0.4.1",
-        "grunt-contrib-clean": "~0.5.0",
-        "grunt-contrib-jshint": "~0.6.4",
-        "grunt-contrib-watch": "~0.5.3",
-        "grunt-dox": "~0.5.0",
-        "grunt-exec": "~0.4.0-next",
-        "grunt-githooks": "~0.3.1",
-        "grunt-github-pages": "~0.0.3",
-        "grunt-gjslint": "~0.1.0",
-        "grunt-mkdir": "~0.1.1",
-        "grunt-mocha-test": "~0.7.0",
-        "grunt-plato": "~0.2.0",
-        "istanbul": "~0.1.34",
-        "mocha": "~1.13.0",
-        "proxyquire": "0.5.1",
-        "should": "^4.0.4",
-        "sinon": "~1.7.3",
-        "sinon-chai": "2.4.0"
-    },
-    "keywords": [],
-    "dependencies": {
-        "async": "^0.9.0",
-        "body-parser": "^1.5.2",
-        "express": "^4.7.2",
-        "mongodb": "^1.4.7",
-        "nodemailer": "^1.1.1",
-        "nodemailer-smtp-transport": "^0.1.12",
-        "request": "^2.39.0",
-        "uuid": "^1.4.1",
-        "logops": "0.2.0"
-    }
+  "name": "perseo",
```

Comment:
Github marks changes in the complete file... is it only related with indentation or is there any actual change?